src/lib/layer-manager.js

```diff
@@ -173,6 +170,26 @@ export default class LayerManager {
     });
   }
 
+  pickLayerByBoundingBox({x, y, width, height}) {
```

Comment:
I assume this function serves as an extension of the pickLayer() function right above it and it will replace pickLayer(), right?